plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/nessie/TestIcebergNessieCatalogConnectorSmokeTest.java

```diff
@@ -140,8 +163,8 @@ protected void dropTableFromMetastore(String tableName)
     @Override
     protected String getMetadataLocation(String tableName)
     {
-        // used when registering a table, which is not supported by the Nessie catalog
-        throw new UnsupportedOperationException("metadata location for register_table is not supported");
+        BaseTable table = (BaseTable) catalog.loadTable(TableIdentifier.of("tpch", tableName));
```

Comment:
`getMetadataLocation` is a required functionality and it is used for other than register table also.